src/main/external-resources/renderers/Panasonic-VieraST60.conf

```diff
@@ -12,11 +12,18 @@ RendererIcon = panasonic.png
 # User-Agent: Panasonic MIL DLNA CP UPnP/1.0 DLNADOC/1.50
 # X-PANASONIC-Registration: VklFUkEgU1Q2MCBTZXJpZXMAAAAAAAAAAAA=
 # friendlyName=VIERA ST60 Series
+#
+# # X-PANASONIC-DMP-Profile=""
+# X-PANASONIC-Registration: VklFUkEgU1Q2MAAAAAAAAAAAAAAAAAAAAAA=
```

Comment:
@Sami32 Sorry, I don't understand what you mean by "later differentiation use". I never thought it was unsafe, I simply wondered if it was useful.